cmd/cloud/user.go

```diff
@@ -0,0 +1,62 @@
+package cloud
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+)
+
+var role string
+
+func newUserCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "user",
+		Aliases: []string{"us"},
+		Short:   "Invite Users to your Astro Organization",
+		Long:    "Invite Users to your Astro Organization.",
+	}
+	cmd.SetOut(out)
+	cmd.AddCommand(
+		newUserInviteCmd(out),
+	)
+	return cmd
+}
+
+func newUserInviteCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "invite",
+		Aliases: []string{"inv"},
+		Short:   "Invite a user to your Astro Organization",
+		Long:    "astro user invite [email] --role [ORGANIZATION_MEMBER, ORGANIZATION_BILLING_ADMIN, ORGANIZATION_OWNER].",
```

Comment:
I think we should give a long description here that elaborates on the short description. This description is good but it could be added to use like:

```
invite [email]
```

maybe to show something similar you could use an example like here: https://github.com/astronomer/astro-cli/blob/0bba7bae7b7eb33df40d977dd45130325050c216/cmd/cloud/deployment.go#L36-L40